docs/server/source/http-client-server-api.rst

```diff
@@ -161,6 +165,25 @@ Transactions
    :statuscode 202: The pushed transaction was accepted in the ``BACKLOG``, but the processing has not been completed.
    :statuscode 400: The transaction was malformed and not accepted in the ``BACKLOG``.
 
+.. http:post:: /api/v1/transactions?mode={mode}
+
+   .. note::
+
+       This option is only available when using BigchainDB with Tendermint.
+
+   Tendermint offers a `broadcast API
+   <http://tendermint.readthedocs.io/projects/tools/en/master/using-tendermint.html#broadcast-api>`_ with three different modes to send transactions.
+   By setting the mode, a new transaction can be pushed with a different mode than the default. The default mode is ``broadcast_tx_async``, which
+   will return immediately and not wait to see if the transaction is valid.
+
+   :query string mode: (Optional) One of the three supported modes to send a transaction: ``broadcast_tx_async``, ``broadcast_tx_sync``, ``broadcast_tx_commit``.
```

Comment:
Shouldn't the possible mode values be `async`, `sync` and `commit` here?

Reply:
oh! You're right! I'll change it.

Reply:
It would be helpful to the reader to summarize what each mode means here, e.g. 

- `async` means post the transaction and return
- `sync` means post the transaction and return after `checkTx` is executed
- `commit` means post the transaction and return after the transaction has been committed